agent/src/main.rs

```diff
@@ -34,19 +40,23 @@ async fn main() {
                 opts.directories_list_args.unwrap_or_default();
             let directories_watch_args: Vec<std::path::PathBuf> =
                 opts.directories_watch_args.unwrap_or_default();
+            debug!("directories_list_args = {:?}", directories_list_args);
+            debug!("directories_watch_args = {:?}", directories_watch_args);
 
             match lister::list_directories(directories_list_args) {
                 Ok(_files_vec) => {
-                    println!("Files: {:?}", _files_vec);
+                    //debug!("{:?}", _files_vec);
```

Comment:
i think we don't need this on main

Reply:
@Bootoyka This PR has nothing to do with main architecture

Reply:
If you want add some debug on a messy main, ok
But don't add it as a commentary, it's useless.
So far, good job we now have a logger